website/content/en/status/report-2025-01-2025-03/syzkaller-wifi.adoc

```diff
@@ -0,0 +1,27 @@
+=== Syzkaller Improvement for WiFi on FreeBSD
+
+Links: +
+link:https://github.com/google/syzkaller[google/syzkaller] URL: link:https://github.com/google/syzkaller[] +
+link:https://github.com/estarriol43/syzkaller/tree/freebsd/frame-injection-v2[work repository] URL: link:https://github.com/estarriol43/syzkaller/tree/freebsd/frame-injection-v2[]
+
+Contact: Jian-Lin Li <ljianlin99@gmail.com> +
+Contact: Li-Wen Hsu <lwhsu@FreeBSD.org>
+
+Syzkaller is an operating system kernel fuzzer that can look for vulnerabilities in the kernel.
+
+This project aims to improve the support of Syzkaller on FreeBSD.
+Based on the existing WiFi fuzzer designed for Linux, we drafted a WiFi fuzzer for FreeBSD.
+We planned to use wtap(4), a virtual wifi interface for testing, in order to support WiFi fuzzing.
```

Comment:
```suggestion
We planned to use man:wtap[4], a virtual wifi interface for testing, in order to support WiFi fuzzing.
```

Reply:
I'll make that change, however, we don't have wtap(4) man page (for now, it's WIP) so it will result for a broken, at least for a while. (I assume that's acceptable.)